.travis.yml

```diff
@@ -1,9 +1,14 @@
 language: ruby
 rvm:
 - 2.1
-- 2.0.0
+- 2.2
+- 2.0
 - 1.9.3
```

Comment:
Yeah I'm trying to work out the matrix exclude so I can skip this test on 1.9.3.